setup/features/script-stage/index.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: single
+title:  "Configuring the Script stage"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+## Purpose
+The [Script stage](/reference/pipeline/stages/#script) lets you run an arbitrary
+shell, Python, or Groovy script on a Jenkins instance as a first class stage in
+Spinnaker. For example, you can use it to launch a test suite from a pipeline
+instead of doing it manually. In order to be able to use the Script stage, you
+need to configure
+
+## Prerequisites
+
+In order to configure a Script stage, you need:
+
+*   A running Jenkins instance at `$JENKINS_HOST`, with a user profile set up
+    with admin access
+*   A running Spinnaker instance, with access to configuration files, and in
+    which you have [set up Jenkins on Spinnaker](/setup/ci/jenkins/)
+
+## Configure Jenkins
+
+1.  `ssh` into your Jenkins machine.
+
+2.  Download the [raw job xml config
+    file](https://storage.googleapis.com/jenkins-script-stage-config/scriptJobConfig.xml)
+    with the command:
+
+    ```bash
+    curl -X GET \
+        -o "scriptJobConfig.xml" \
+        "https://storage.googleapis.com/jenkins-script-stage-config/scriptJobConfig.xml"
+    ```
+
+3.  Create the Jenkins job where your script will run. To do this, you need the
```

Comment:
Be mindful of using "this." Studies show uses don't really know what we're referring to. Perhaps better to say: "To create the job, you need..."